test/DurableOrchestrationClientMock.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
```

Comment:
All source files need to include the copyright header.

Reply:
Great point. I missed this element.